lib/components/writer/etcher.js

```diff
@@ -48,13 +49,33 @@ exports.writeImage = async (image, destination) => {
     check: true
   })
 
-  emitter.on('progress', (state) => {
-    console.log(`Flashing ${image}: ${state.percentage.toFixed(2)}% (${state.type})`)
+  const bar = new visuals.Progress('')
+
+  let report = (data) => {
+    if (Math.floor(data.percentage) % 5 === 0) {
```

Comment:
So this is basically the same logic we have on the SDK. Should we create shared utility function to "print" progress, that decides what to do depending on the `CI` variable?